pages/api/index.js

```diff
@@ -10,6 +12,14 @@ function API({url, content, locale}) {
       locale={locale}
       contributors={content.contributors}
     >
+      <Head>
+        {twitterCard({
+          card: 'summary',
+          title: content.title,
```

Comment:
The `tagline` isn't the best title here, IMO. I should be more like `glamorous - ${tagline}`, so we get something like:
`glamorous - Basic Concepts`

Or we can do `Basic Concepts for glamorous`, `Basic Concepts | glamorous`, `glamorous | Basic Concepts`, etc. You get the idea.